README.md

```diff
@@ -1,21 +1,31 @@
+r# Rule Command Line Interface Documentation
```

Comment:
The heading # is prefaced with an errant 'r' which is why the heading doesn't render properly